.github/workflows/unit-tests.yml

```diff
@@ -39,18 +39,9 @@ jobs:
     strategy:
       matrix:
         include:
-         - solr: 6.6.5
-           db: mysql
          - solr: 8.6
            db: postgresql
```

Comment:
We should get rid of the matrix entirely since we only want to run one build. And we can go back to using the `.python-version` file in the project to manage the version for the build, which should help with the upgrade. The geniza unit tests have an example of how we're doing that.